--- kdebase-4.5.1/konsole/src/Screen.cpp~	2010-04-28 10:01:02.000000000 +0200
+++ kdebase-4.5.1/konsole/src/Screen.cpp	2010-09-27 09:23:48.658050544 +0200
@@ -1223,6 +1223,19 @@
         Character* data = screenLines[screenLine].data();
         int length = screenLines[screenLine].count();
 
+	// Don't remove end spaces in lines that wrap
+	if (!(lineProperties[screenLine] & LINE_WRAPPED))
+	{
+		 // ignore trailing white space at the end of the line
+		 for (int i = length-1; i >= 0; i--)
+		 {
+			 if (data[i].character == ' ')
+				 length--;
+			 else
+				 break;
+		 }
+	}
+
         //retrieve line from screen image
         for (int i=start;i < qMin(start+count,length);i++)
         {
